aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -1018,6 +1018,23 @@ Tensor& heaviside_(Tensor& self, const Tensor& values) {
   return at::heaviside_out(self, self, values);
 }
 
+Tensor& ldexp_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  TORCH_CHECK(!self.is_complex(), "ldexp does not support complex tensors.");
+  TORCH_CHECK(result.dtype() == self.dtype(), "ldexp result dtype must match input dtype.");
+  native::mul_out(result, self, at::pow(2.0, other));
```

Comment:
I think you may need at::mul_out instead of native here if result is undefined.

Reply:
Also (speaking of little used functions), you can use exp2 here: https://pytorch.org/docs/master/generated/torch.exp2.html?highlight=exp2#torch.exp2.

Reply:
re: exp2 - I hit an issue there where it couldn't handle Long data types.

Reply:
Oh that's a good point - it probably doesn't implement dispatch for long properly. Let's leave it as is, then. Thanks for following up, @ranman